mitodl/open-discussions
# search page

#### Pre-Flight checklist

<img width="1673" alt="Screen Shot 2020-04-28 at 10 00 39 AM" src="https://user-images.githubusercontent.com/1934992/80496400-28446900-8937-11ea-939e-50c4ce362fe7.png">

<img width="357" alt="Screen Shot 2020-04-28 at 9 45 41 AM" src="https://user-images.githubusercontent.com/1934992/80494861-2e394a80-8935-11ea-90f3-65a63a5eac0d.png">

#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/2743

#### What's this PR do?
This PR adds podcasts and podcast episodes as learning resources in the search page

#### How should this be manually tested?
Import podcasts and podcast episodes with
`docker-compose run web ./manage.py backpopulate_podcast_data`

Go to `http://localhost:8063/learn/search` 
- Podcasts and podcast episodes should show up  as filterable learning resources in the sidebar
- Podcasts and podcast episodes should show up in search results
- Adding and removing podcasts from favorites and lists should work
- Playing audio should work in the search page
- There should not be regressions in the podcasts page
